advocacy_docs/partner_docs/Cyberark/04-Configuratingcyberark.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Configuration'
+description: 'Walkthrough on configuring the integration'
+---
+
+## Prerequisites
+
+Implementing CyberArk CPM (Central Policy Manager) with EDB Postgres Advanced Server requires the following components:
```

Comment:
There are no components listed after the colon.

Reply:
Removed "components".